packages/components/addon/components/hds/app-footer/copyright.hbs

```diff
@@ -0,0 +1,9 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class="hds-app-footer__copyright" ...attributes>
+  <FlightIcon @name="hashicorp" />
```

Comment:
I think @andgen404 is changing this in his component, but this should probably be `hashicorp-color`, is that right?

Reply:
If we're going to implement the "dark" version, is better to tint it dynamically (so should not be the "color" version).